src/medley/core.cljc

```diff
@@ -658,3 +658,11 @@
   {:added "1.4.0"}
   [x]
   (instance? #?(:clj java.util.regex.Pattern :cljs js/RegExp) x))
+
+(defn index-of
+  "Returns the index of the first occurrence of x in a sequential collection, coll,
```

Comment:
Docstring says `coll` but param is called `xs` so need to make them agree, one way or the other.

Reply:
Ah, yep - `coll` is preferred in this case, since it tends to be what's used in many `clojure.core` functions.